<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriangleDialog</class>
 <widget class="QDialog" name="TriangleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>139</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите параметры</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrushColor">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>150</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать цвет заливки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PenColor">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>190</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать цвет контура</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="position_Y">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>80</y>
     <width>136</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="side_size">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>47</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>47</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сторона</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="base_size">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>80</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="position_X">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>47</y>
     <width>136</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>81</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Основание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TriangleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TriangleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
